test/test_namedtuple_return_api.py

```diff
@@ -86,10 +86,7 @@ def test_namedtuple_return(self):
         '''))
 
         # test pstrf
-        b = torch.mm(a, a.t())
-        # add a small number to the diagonal to make the matrix numerically positive semidefinite
-        for i in range(a.size(0)):
-            b[i][i] = b[i][i] + 1e-7
+        b = torch.eye(5)
         ret = b.pstrf()
         self.assertIs(ret.u, ret[0])
         self.assertIs(ret.pivot, ret[1])
```

Comment:
I'm not sure what `u` and `pivot` are but as long as they're not the same, this test should be good.

Reply:
I don't think they are the same:
> `u` is upper triangular such that ....., with p the permutation given by `pivot`